tests/e2e/scale/test_scale_pgsql.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import pytest
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.scale_pgsql import ScalePodPGSQL
+from ocs_ci.utility import utils
+from ocs_ci.framework.testlib import (
+    E2ETest, scale, ignore_leftovers
+)
+from ocs_ci.ocs.node import get_node_resource_utilization_from_adm_top
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def pgsql(request):
+
+    pgsql = ScalePodPGSQL()
+
+    def teardown():
+        pgsql.cleanup()
+    request.addfinalizer(teardown)
+    return pgsql
+
+
+@scale
+@ignore_leftovers
+@pytest.mark.polarion_id("OCS-2239")
+class TestPgsqlPodScale(E2ETest):
+    """
+    Scale test case using PGSQL Pods
+    """
+
+    def test_scale_pgsql(self, pgsql):
+        """
+        Test case to scale pgsql pods:
+          * Add worker nodes to existing cluster
+          * Label new worker node
+          * Create pgsql to run on 200 pods on new added worker node
+        """
+        replicas = 100  # Number of postgres and pgbench pods to be deployed
+        timeout = replicas * 500  # Time in seconds to wait for pgbench pods to be created
```

Comment:
does it take ~13 hours? I see that the test has got completed in 1.5 hrs, so why such a long timeout value?

Reply:
Since PVC creation limitation is fixed (https://bugzilla.redhat.com/show_bug.cgi?id=1836198),  I'm going to increase the number of pgbench pods and adjust the time out value.